substrate/frame/multisig/src/tests.rs

```diff
@@ -20,18 +20,13 @@
 #![cfg(test)]
 
 use super::*;
-
 use crate as pallet_multisig;
-use frame_support::{
-	assert_noop, assert_ok, derive_impl,
-	traits::{ConstU32, ConstU64, Contains},
-};
-use sp_runtime::{BuildStorage, TokenError};
+use frame::{prelude::*, runtime::prelude::*, testing_prelude::*};
```

Comment:
One little detail here is which preludes are a superset of the others. 

I recall I made a deliberate choice about preludes being a superset of one another, but I didn't update the code 🙈 For example here, I think `testing_prelude` brings in the other two